draft-ietf-ppm-dap.md

```diff
@@ -1457,9 +1465,9 @@ Next, the Helper checks that the report IDs in
 initialization messages have the same report ID, then the Helper MUST abort with
 error `invalidMessage`.
 
-If the Helper finds the `AggregationJobInitReq` to be valid, it immediately
-responds with HTTP status 201 Created. The Helper is now ready to process
-the aggregation job, asynchronously from any further requests from the leader.
+If the Helper finds the `AggregationJobInitReq` to be valid, it MAY immediately
+respond with HTTP status 201 Created with no body, and perform the remaining
```

Comment:
Mention media type here as well.

Reply:
I think it may be better if I strike `with no body`, as elsewhere in collection it's inferred that if we only say "responds with STATUS" there is no media type and body, as compared to explicitly saying "responds with STATUS with a body of SomeResp and media type foo/bar"